inspire/modules/workflows/static/js/workflows/actions/arxiv_approval.js

```diff
@@ -102,20 +102,22 @@ define(
         var $this = this;
         var objectids = payload.objectids;
 
-        jQuery.ajax({
-          type: "POST",
-          url: $this.attr.action_url,
-          data: payload,
-          success: function(data) {
-            $this.post_request(data);
-            $this.trigger(document, "removeSentElements", {"ids":objectids});
-          },
-          error: function(request, status, error) {
-            console.log(error);
-          }
-        });
+        if (objectids) {
```

Comment:
Is `objectids` usually an empty array? If so, traditionally I think this check only returns `false` for (empty string), false, null, undefined, 0, NaN but not (empty array) [].

Probably safer to change to: `objectsids.length > 0` if this variable is guaranteed to be an array.